tests/test_value_parser.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 Prediktor AS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opcua_tools.value_parser import parse_boolean
+import pytest
+import pandas as pd
+
+
+@pytest.fixture(scope="session")
+def test_data():
+    list = [
+        ["Boolean", "false", 0],
+        ["Boolean", "False", 0],
```

Comment:
Perhaps using False and True instead of numbers is clearer? If it works :-)

Reply:
Haha you are absolutely right, then I can skip a mapping step aswell. I will just push this minor change directly to the main branch if permissions allow it.